vidarrtest-regression.json.in

```diff
@@ -68,6 +68,8 @@
                 }
             ],
             "starFusion.runStarFusion.genomeDir": null,
+            "starFusion.reference": "hg38",
+            "starFusion.outputFilePrefix": "STAR-Fusion_golden_data",
             "starFusion.runStarFusion.jobMemory": null,
             "starFusion.runStarFusion.modules": null,
             "starFusion.runStarFusion.starFusion": null,
```

Comment:
**modules**, **starFusion** and **genomeDir** should not be in RT

Reply:
done